ci/scripts/check-ascii.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
Is it bad that I think this is a bit too complicated and maybe should just be a dumb python script